client/trino-client/src/main/java/io/trino/client/auth/kerberos/DelegatedUnconstrainedContextProvider.java

```diff
@@ -21,18 +21,18 @@
 import javax.security.auth.kerberos.KerberosTicket;
 import javax.security.auth.login.LoginException;
 
+import java.security.AccessController;
 import java.util.Set;
 
 import static com.google.common.collect.Iterables.getOnlyElement;
-import static java.security.AccessController.getContext;
 
-public class ContextBasedSubjectProvider
-        implements SubjectProvider
+public class DelegatedUnconstrainedContextProvider
```

Comment:
But both the approaches are `Unconstrained` right ?

Reply:
renamed